File: Python/Array/Union of Arrays with Duplicates.py
//https://www.geeksforgeeks.org/problems/union-of-two-arrays3538/1
class Solution:    
    # Function to return the count of number of elements in union of two arrays.
    def findUnion(self, a, b):
        s = set()
        for num in a:
            s.add(num)
        for num in b:
            s.add(num)
        return len(s)

File: Cpp/Array/Union of Arrays with Duplicates.cpp
//https://www.geeksforgeeks.org/problems/union-of-two-arrays3538/1
class Solution{
    public:
    //Function to return the count of number of elements in union of two arrays.
    int doUnion(int a[], int n, int b[], int m)  {
        //code here
       set<int> s;
       for(int i=0;i<n;i++){
           s.insert(a[i]);
       }
       for(int j=0;j<m;j++){
           s.insert(b[j]);
       }
       return s.size();
    }
};
